include/client/new/types.h

```diff
@@ -437,12 +463,17 @@ namespace [[cheerp::genericjs]] client {
 		void set_caller(Function* caller);
 		template<class... _Args>
 		Function(_Args... args);
+		template<class... _Args>
+		[[gnu::always_inline]]
+		static Function* _New(_Args... args) {
+			return new Function(args...);
+		}
 		static Function* prototype;
 	protected:
 		[[cheerp::client_transparent]]
 		Function(EventListener* listener) noexcept;
 		[[cheerp::client_transparent]]
-		Function(const EventListener* listener) noexcept;
```

Comment:
is dropping of noexcept intended?

Reply:
No, it was not intended.

This was the result of a mistake in an earlier commit that leaves me wondering how the `noexcept` was ever there in the first place.
![image](https://github.com/leaningtech/cheerp-utils/assets/6209323/fb62350c-6ae0-44e5-8a81-b8d7c5d1b1a8)

Regardless, it's fixed now.